services/ingestion/engine.go

```diff
@@ -151,6 +155,7 @@ func (e *Engine) processEvents(events *models.CadenceEvents) error {
 	if err != nil {
 		return fmt.Errorf("failed to index block %d event: %w", events.Block().Height, err)
 	}
+	e.collector.EvmBlockIndexed()
```

Comment:
this should go at the end of the function, and it should take in the EVM height which was indexed